qml/Launcher/LauncherDelegate.qml

```diff
@@ -37,7 +37,7 @@ Item {
 
     readonly property int effectiveHeight: Math.cos(angle * Math.PI / 180) * itemHeight
     readonly property real foldedHeight: Math.cos(maxAngle * Math.PI / 180) * itemHeight
-    readonly property alias wiggling: wiggleAnim.running
+    readonly property alias wiggling: root.alerting
```

Comment:
Does the old code cause any problems? It seems to be written this way to make sure this property is changed after the animation is actually running. See https://github.com/ubports/unity8/commit/e46dc1dc552f1ddbc27de581b49ba5dfda877db4.

Reply:
IIRC, it was a binding loop, since `wiggleAnim.running` just points at `alerting`.

Reply:
No, but the test does fail if the old value is used. I think it should be using a SignalSpy anyway, so I'll get that set up...

Reply:
Nope! There's actually a bug here. Wiggling only starts on the first time the count becomes visible. `alerting` is never cleared so the wiggle can start the second time.

Reply:
It appears that the normal LauncherModel never clears the `alerting` property when the count becomes invisible, either. I think this is expected behavior and the test should be resetting the property between runs.

Reply:
All of those hypotheses were wrong. The problem is actually that we're using [alwaysRunToEnd](https://doc.qt.io/qt-5/qml-qtquick-animation.html#alwaysRunToEnd-prop), meaning the animation can be stopped (which sets `running` to `false`) before it actually completes and visually stops.

Reply:
Addressed in https://github.com/ubports/unity8/pull/355/commits/627bd76e70ff6e3d5ab427a7103d8e0039be8c2e